rwengine/src/audio/SoundSource.cpp

```diff
@@ -377,10 +372,31 @@ void SoundSource::decodeAndResampleFrames(const std::filesystem::path& filePath,
             while ((receiveFrame =
                         avcodec_receive_frame(codecContext, frame)) == 0) {
                 if (!swr) {
+#if !HAVE_CH_LAYOUT
```

Comment:
Here you're doing
```c
#if !HAVE_CH_LAYOUT
...
#endif
#if HAVE_CH_LAYOUT
...
#else
```
which could be baked together into a single `#if` as far as I can tell.